src/Uno.UI.RuntimeTests/UnitTestsImport.props

```diff
@@ -1,6 +1,30 @@
 ﻿<Project>
 
 
+  <ItemGroup>
+	<Compile Include="$(MSBuildThisFileDirectory)..\Uno.UI.Tests\Windows_Globalization\Given_SignificantDigitsNumberRounder.cs">
```

Comment:
Consider adding these back to the original location to avoid diff noise and make things easier to review.